tests/test_layered_image.py

```diff
@@ -169,6 +169,24 @@ def test_grow_mask(self):
                            (x == 11 and y <= 13 and y >= 11))
             self.assertEqual(science.pixel_has_data(x, y), not should_mask)
 
+   def test_grow_mask_mult(self):
+      mask = self.image.get_mask()
+      mask.set_pixel(10, 11, 1)
+      mask.set_pixel(10, 12, 1)
+      self.image.set_mask(mask)
+      self.image.apply_mask_flags(1, [])
+      self.image.grow_mask(3)
+
+      # Check that the mask has grown to all applicable pixels.
+      science = self.image.get_science()
+      for y in range(self.image.get_height()):
+         for x in range(self.image.get_width()):
+            # Check whether the point is a manhattan distance of <= 3 from
+            # one of the original masked pixels.
+            dx = abs(x - 10)
+            dy = min(abs(y - 11), abs(y - 12))
```

Comment:
minor/optional - this is a clever way to test the masking, but if the values 3, [10, 11], [10, 12] were variables it might be a little easier to read.